examples/misc/example_template.py

```diff
@@ -44,8 +45,8 @@
 
 # Your code goes here...
 mesh_util = prime.lucid.Mesh(model=model)
-example_file = prime.examples.download_elbow_fmd()
-mesh_util.read(example_file)
+mixing_elbow = prime.examples.get_file(Examples.ELBOW_PMDAT)
```

Comment:
@AlejandroFernandezLuces 
```suggestion
mixing_elbow = prime.examples.get_file(Examples.ELBOW_FMD)
```